pkg/objectstore/azure/azure.go

```diff
@@ -24,14 +30,46 @@ func getPipeline(backupLocation *stork_api.BackupLocation) (pipeline.Pipeline, e
 	return azureblob.NewPipeline(credential, azblob.PipelineOptions{}), nil
 }
 
+func getAzureURLSuffix(backupLocation *stork_api.BackupLocation) (string, error) {
+	// Give first preference to environment variable setting.
+	azureEnv := os.Getenv(azureEnvKey)
+	if len(azureEnv) == 0 {
+		// If env variable is not set, check the azure environment vaue from backuplocation CR.
+		if len(backupLocation.Location.AzureConfig.Environment) != 0 {
+			azureEnv = backupLocation.Location.AzureConfig.Environment
+		}
+	}
+	// If both env and CR value are empty, set it default as public type.
+	if len(azureEnv) == 0 {
+		return az_autorest.PublicCloud.StorageEndpointSuffix, nil
+	}
+	azureClientEnv, err := az_autorest.EnvironmentFromName(backupLocation.Location.AzureConfig.Environment)
+	if err != nil {
```

Comment:
you can avoid this err check as you are already checking this at the line 62 below. You can move the logrus.Errorf() there itself. too many err checks.

Reply:
if you avoid the error check, our `go errcheck` would fail during compilation. At every place where error is generated, we need to have error check done.